docs/source/openstack/volume-migration-using-s3.md

```diff
@@ -0,0 +1,190 @@
+|           |   public: https://kzn-swift.massopen.cloud/swift/v1   |
+|           | moc-kzn                                               |
+|           |   internal: https://kzn-swift.massopen.cloud/swift/v1 |
+|           |                                                       |
++-----------+-------------------------------------------------------+
+```
+
+And from the NERC:
+
+```
+$ openstack --os-cloud nerc catalog show object-store -c endpoints
++-----------+----------------------------------------------------------------------------------------+
+| Field     | Value                                                                                  |
++-----------+----------------------------------------------------------------------------------------+
+| endpoints | regionOne                                                                              |
+|           |   admin: http://10.255.2.253:8080                                                      |
+|           | regionOne                                                                              |
+|           |   internal: http://10.255.2.253:8080/v1/AUTH_01234567890123456789012345689012          |
+|           | regionOne                                                                              |
+|           |   public: https://stack.nerc.mghpcc.org:13808/v1/AUTH_01234567890123456789012345689012 |
+|           |                                                                                        |
++-----------+----------------------------------------------------------------------------------------+
+```
+
+## Configure minio client
+
+For each environment, create a MinIO alias configuration using the base URL of the "public" interface of the object-store service and the access key and secret your generated earlier:
+
+```
+$ mc alias set kaizen https://kzn-swift.massopen.cloud 1234567890ABCDEF1234567890ABCDEF 1234567890ABCDEF1234567890ABCDEF
+$ mc alias set nerc https://stack.nerc.mghpcc.org:13808 FEDCBA0987654321FEDCBA0987654321 FEDCBA0987654321FEDCBA0987654321
+```
+
+## Create volume backup in source environment
+
+Create a volume backup in kaizen:
+
+```
+$ openstack --os-cloud kaizen volume backup create --container <bucket_name> <volume id or name>
+```
+
+In the above command line, `<bucket_name>` is the name of the object storage container that will hold your backup. This must be unique across all users, so using your username as a prefix is probably a good idea (e.g., `--container larsks-backups`).
+
+This will create a set of objects in the named bucket. You can use the MinIO client to inspect it:
+
+```
+$ mcli ls kaizen
+[2022-03-17 13:48:47 EDT]     0B larsks-backups/
+$ mcli ls kaizen/larsks-backups/
+[2022-03-21 20:02:55 EDT]     0B volume_bb2086dc-d6a4-4346-9809-f47fbde673f3/
+```
+
+## Create target container in destination environment
+
+We need to create a bucket in the target environment to receive the backup:
+
+```
+$ mcli mb nerc/larsks-backups
```

Comment:
this might be a windows thing but once again `~/Downloads/mc` works `mcli` does not.